### DISCUSSION TITLE
Redirecting to admin

### DISCUSSION THREAD

**Discussion by ogurec-ogurec** [danirus/django-comments-xtd#247] [Q&A] [answered]
Hello, I have a little problem. If I click the like button for an unauthorized user, I am redirected to the admin page:

`localhost/admin/login/?next=/comments/like/290/`

my settings: 

```
COMMENTS_XTD_APP_MODEL_OPTIONS = {
    'blog.post': {
        'allow_flagging': True,
        'allow_feedback': True,
        'show_feedback': True,
        'who_can_post': 'users'
    }
}
```

How do I make it redirect to the `/login` page (not the admin panel)?

**Comment by danirus** [comment, marked as answer]
Hi @ogurec-ogurec, there is a setting for this in Django: [settings.LOGIN_URL](https://docs.djangoproject.com/en/3.1/ref/settings/#login-url). 
In the comp demo site I use the login URL of the admin site. 
Changing the LOGIN_URL you can redirect to your own.

**Reply by ogurec-ogurec** [reply]
have the settings:

```
LOGIN_URL = '/login/'
LOGIN_REDIRECT_URL = '/login/'
```

And the page does redirect to the login, only the url in the browser bar looks like:

`site.ru/admin/login/?next =/comments/like/291/`

I thought that this address might scare the user, and would like something like `site.ru/login/?next =/comments/like/291/`
without the word admin :)

Seems to have found: 

`\Lib\site-packages\django_comments_xtd\api\frontend.py:114`:

` d['login_url'] = "/login/"`

Thanks you!
